rewrite-core/src/main/java/org/openrewrite/config/ClasspathScanningLoader.java

```diff
@@ -170,7 +175,7 @@ private void configureRecipes(ScanResult result, String className) {
             Timer.Builder builder = Timer.builder("rewrite.scan.configure.recipe");
             Timer.Sample sample = Timer.start();
             try {
-                Recipe recipe = constructRecipe(recipeClass);
+                Recipe recipe = constructRecipe(recipeClass).augmentRecipeDescriptor(artifactManifestAttributes.get(classInfo.getClasspathElementURI().toString()));
```

Comment:
I see your Idea here for a fluent API. But ATM all other data is set by calling the respective `set` methode. We should stick to this pattern for now.